src/components/views/elements/AppTile.tsx

```diff
@@ -179,7 +179,8 @@ export default class AppTile extends React.Component<IProps, IState> {
         if (this.props.room.roomId == RoomViewStore.getRoomId()) return;
         const app = this.props.app;
         const isActiveWidget = ActiveWidgetStore.instance.getWidgetPersistence(app.id);
-        if (!isActiveWidget) {
+        // Stop the widget if it's not the active (persistent) widget and it's not a user widget
+        if (!isActiveWidget && app.roomId !== undefined) {
```

Comment:
It's somewhat worrying that `app.roomId`, `props.room`, and `props.userWidget` are all possible ways to check if something is a user widget... From a quick glance, I'm unsure if they all mean the same thing. 😓

Reply:
ah, props.userWidget is probably better